### PR TITLE
Created the launch announcement blog post for Data Prepper 1.5.0.

### PR DESCRIPTION
**Pull request by dlvenable**
### Description

Blog post announcing the launch of Data Prepper 1.5.0, focused on the S3 Source support.
 
### Issues Resolved

N/A

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by krisfreedain** [comment]
looks great from my side - having editorial team take a look.

**Review by natebower — CHANGES_REQUESTED** [review]
@dlvenable Nice job on this. Please see my comments, and let me know if you have any questions. I will review and approve once the changes are implemented. Thanks!

File: _posts/2022-06-23-S3-Log-Ingestion-Using-Data-Prepper-1.5.0.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+notifications to configured destinations whenever objects in an S3 bucket change. For example, if a new object is 
+created, S3 will send this notification. You can configure an S3 bucket to send event notifications to an 
+[Amazon SQS](https://aws.amazon.com/sqs/) queue whenever new objects are written to S3. You then configure Data Prepper 
+to use that SQS queue for receiving event notifications.
+
+Data Prepper will receive messages from the SQS queue. For any newly created object, it then gets that object out of S3 
+and parses it into events. Initially, Data Prepper can read two types of formats.
+
+1. Single-line logs. These are logs where a single log line indicates the same event.
+2. JSON objects. Data Prepper expects a common JSON pattern where the JSON has one large JSON array of smaller objects. Data Prepper will take each smaller object and create a single event from that object.
```
[the remark this answers is not in the document]
Comment (dlvenable) [reply _posts/2022-06-23-S3-Log-Ingestion-Using-Data-Prepper-1.5.0.md:48]: I wasn't sure what the right noun was here. I could be "object" but I didn't want to cause confusion with object later in the sentence. How about "JSON structure"?

File: _posts/2022-06-23-S3-Log-Ingestion-Using-Data-Prepper-1.5.0.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+Additionally, Data Prepper supports either uncompressed data or Gzip-compressed data.
+
+
+### An Example
+
+Earlier, I mentioned supporting Application Load Balancer (ALB) access logs. These logs are saved as traditional logs.
```
[the remark this answers is not in the document]
Comment (dlvenable) [reply _posts/2022-06-23-S3-Log-Ingestion-Using-Data-Prepper-1.5.0.md:55]: I use "ALB" in the next sentence. Should I remove it still?

File: _posts/2022-06-23-S3-Log-Ingestion-Using-Data-Prepper-1.5.0.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+- dlv
+date: 2022-06-23 12:00:00 -0500
+categories:
+  - technical-post
+twittercard:
+  description: "You can start using Data Prepper 1.5.0 to ingest log data from S3 today."
+---
+
+Data Prepper is an open-source data collector for data ingestion into OpenSearch. It currently supports trace analytics 
+and log analysis use-cases. Earlier this year Data Prepper added log ingestion over HTTP using tools such as
```
[the remark this answers is not in the document]
Comment (dlvenable) [reply _posts/2022-06-23-S3-Log-Ingestion-Using-Data-Prepper-1.5.0.md]: Thanks @cmanning09! I added a new sentence about this.

**Comment by dlvenable** [comment]
@natebower, I believe I incorporated most of your feedback.

The one part I didn't change was dropping "ALB." This seemed a little too wordy to me. Is there a particular reason to not include "ALB?"

File: _posts/2022-06-23-S3-Log-Ingestion-Using-Data-Prepper-1.5.0.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (natebower) [reply _posts/2022-06-23-S3-Log-Ingestion-Using-Data-Prepper-1.5.0.md:56]: We should almost always spell out Application Load Balancer. I know it may seem wordy, but it's the proper name of the resource and is what we use in the AWS documentation. On the consoles, when space is limited (such as dropdown lists), it’s acceptable to use the ALB acronym if the context is clear.

File: _posts/2022-06-23-S3-Log-Ingestion-Using-Data-Prepper-1.5.0.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+### An example
+
+Earlier, I mentioned supporting Application Load Balancer (ALB) access logs. These logs are saved as traditional logs. 
+Each network request is a single line in the log file, and that line follows a specific format. Additionally, ALB logs are stored with 
+gzip compression in S3.
+
+There is a lot of rich information in these logs. You can determine whether requests are HTTP, HTTPS, or gRPC, and both the time that 
+the entire request took to process and the time your application took to process it are available. Also, you can get 
+the final status code, User-Agent headers, and AWS X-Ray trace information. Much of this information is also available
```
Comment (natebower) [inline _posts/2022-06-23-S3-Log-Ingestion-Using-Data-Prepper-1.5.0.md:62]: Remove the first "also" in the last sentence.

**Review by natebower — COMMENTED** [review]
@dlvenable Just a couple more comments for you.

**Review by natebower — APPROVED** [review]
LGTM!

**Review by krisfreedain — APPROVED** [review]
Looks good!
